README.md

```diff
@@ -85,11 +109,15 @@ var obj = {
 
 -- YOUR ANSWER HERE --
 
+- ES5
+- no
+- can't be called in the form written
+
```

Comment:
Valid ES6. See [Method Properties](http://es6-features.org/#MethodProperties)

It's called as ```obj.add(33,99);```